desktop/core/src/desktop/templates/global_js_constants.mako

```diff
@@ -136,9 +136,11 @@
   window.JB_SINGLE_CHECK_INTERVAL_IN_MILLIS = 5000;
   window.JB_MULTI_CHECK_INTERVAL_IN_MILLIS = 20000;
   window.CLOSE_SESSIONS = {'hive': '${ CLOSE_SESSIONS.get() }' === 'True'};
+  window.sourceType_phoenix = '${[interpreter['type'] for interpreter in get_ordered_interpreters(request.user) if interpreter['dialect'] == 'phoenix'][0]}';
```

Comment:
Why is a global constant needed?

See hueConfig.ts, for instance findEditorConnector(connector => connector.dialect === 'phoenix')

Reply:
I should also note that we're working on getting rid of the .mako template code so let's not add additional logic here. hueConfig.ts has the necessary functions.

Reply:
done